bindings/http/http.go

```diff
@@ -73,7 +73,7 @@ func (h *HTTPSource) Init(metadata bindings.Metadata) error {
 		return err
 	}
 	var tlsConfig *tls.Config
-	if h.metadata.MTLSEnable == "true" {
+	if h.metadata.MTLSClientCert != "" && h.metadata.MTLSClientKey != "" && h.metadata.MTLSRootCA != "" {
```

Comment:
Is the `rootCA` actually required? If you don't pass a CA cert, you can skip creating a cert pool, and the app will use the system's default cert pool (the CA trust store) to validate certificates. It should be a valid (albeit possibly uncommon) scenario

Reply:
agree, this might be a possible scenario.